modules/nodes-pods-understanding-requests-limits.adoc

```diff
@@ -10,11 +10,11 @@ You can specify CPU and memory requests and limits for pods in pod specs, as sho
 
 CPU and memory _requests_ specify the minimum amount of a resource that a pod needs to run, helping to schedule pods on nodes with sufficient resources. 
 
-CPU and memory _limits_ define the maximum amount of memory a pod can consume, preventing it from consuming excessive memory and potentially impacting other pods on the same node. 
+CPU and memory _limits_ define the maximum amount of a resource that a pod can consume, preventing it from consuming excessive resources and potentially impacting other pods on the same node. 
 
 CPU and memory requests and limits are processed by using the following principles.
 
-* CPU limits are enforced by CPU throttling. When a container approaches its CPU limit, the kernel restricts access to the CPU corresponding to the container's limit. As such, a CPU limit is a hard limit that the kernel enforces. Containers cannot use more CPU than is specified in their CPU limit.
+* CPU limits are enforced by CPU throttling. When a container approaches its CPU limit, the kernel restricts access to the CPU corresponding to the container's limit. As such, a CPU limit is a hard limit that the kernel enforces. A A container might be allowed to exceed its CPU limit for extended periods of time. However, container runtimes do not terminate pods or containers for excessive CPU usage.
```

Comment:
> A container might or might not be allowed to exceed its CPU limit

Fixed in the following [commit](https://github.com/openshift/openshift-docs/pull/91091/commits/fbd31d10ed1fe8428c6d6453bf7d4bef7259223f).